src/test/java/com/systelab/seed/patient/PatientResourceTest.java

```diff
@@ -41,10 +44,19 @@ private Patient getPatientData(String name, String surname, String email) {
         return patient;
     }
 
-    @Description("Create a new patient with name, surname and email")
+    private Patient getPatientData(String name, String surname, String email, String medicalNumber) {
```

Comment:
The getPatientData with 4 arguments should replace the previous getPatientData with only 3. Additionally I would add the parameters that are not being used DOB, Street, City, Zip Code

Reply:
done